resources/version.go

```diff
@@ -48,6 +51,55 @@ func CreateVersion(db *gorm.DB, version *Version) (bool, error) {
 	return result.RowsAffected == 1, result.Error
 }
 
+// versionDeploymentDefinition builds a deployment spec for the provided version
+func versionDeploymentDefinition(version *Version) *v1beta1.Deployment {
+	replicas := int32(version.Replicas)
+	name := fmt.Sprintf(
+		"%v-%v-%v",
+		version.Environment.Application.Slug,
+		version.Environment.Slug,
+		version.Slug,
+	)
+
+	return &v1beta1.Deployment{
+		ObjectMeta: v1.ObjectMeta{
+			Name:      name,
+			Namespace: "brizo",
+			Labels: map[string]string{
+				"brizoManaged": "true",
```

Comment:
Could probably remove the `brizoManaged` if we want to do that. If a resource has a label of `brizoVersion` at all we can safely assume it's managed